src/data-generator/users.ts

```diff
@@ -192,6 +193,89 @@ export async function getUsers(token: string, locationId: string) {
   return res.data.searchUsers.results
 }
 
+export async function getUserByRole(token: string, role: string) {
+  const getUsersRes = await fetch(GATEWAY_GQL_HOST, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${token}`,
+      'x-correlation-id': `getusers`
+    },
+    body: JSON.stringify({
+      operationName: null,
+      variables: {
+        role: role,
+        count: 100
+      },
+      query: `
+        query ($count: Int, $skip: Int, $role: String) {
+          searchUsers(count: $count, skip: $skip, role: $role) {
+            totalItems
+            results {
+              id
+              primaryOffice {
+                id
+              }
+              username
+              role
+            }
+          }
+        }
+      `
+    })
+  })
+
+  const res = (await getUsersRes.json()) as {
+    data: {
+      searchUsers: {
+        results: Array<{
+          username: string
+          role: Role
+          type: string
+          primaryOffice: { id: string }
+        }>
+      }
+    }
+  }
+
+  return res.data.searchUsers.results
+}
+
+const createRegisterSystemUser = async(name: string, type:string, token: string) => {
```

Comment:
Just one small change otherwise it's looking good!
```suggestion
const registerSystemClient = async(name: string, type:string, token: string) => {
```